agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/linkml/DiseaseAnnotationIndexer.java

```diff
@@ -0,0 +1,51 @@
+package org.alliancegenome.indexer.indexers.linkml;
+
+import org.alliancegenome.core.config.ConfigHelper;
+import org.alliancegenome.curation_api.model.entities.AGMDiseaseAnnotation;
+import org.alliancegenome.curation_api.model.entities.AlleleDiseaseAnnotation;
+import org.alliancegenome.curation_api.model.entities.GeneDiseaseAnnotation;
+import org.alliancegenome.curation_api.response.SearchResponse;
+import org.alliancegenome.indexer.DiseaseAnnotationRESTInterface;
+import org.alliancegenome.indexer.RestConfig;
+import org.jboss.logging.Logger;
+import si.mazi.rescu.RestProxyFactory;
+
+
+public class DiseaseAnnotationIndexer {
+
+    private Logger log = Logger.getLogger(getClass());
+    private DiseaseAnnotationRESTInterface api = RestProxyFactory.createProxy(DiseaseAnnotationRESTInterface.class, ConfigHelper.getCurationApiBaseUrl(), RestConfig.config);
```

Comment:
Switch out the RestConfig.config for one that adds the auth header or not depending if its set.